src/rulesets/CodeCop.ruleset.json

```diff
@@ -556,7 +556,8 @@
                   },
                   {
                       "id":  "AA0139",
-                      "action":  "Error"
+                      "action":  "None",
```

Comment:
I should also note:
Changing the rule here, in CodeCop.ruleset,json means the rule will never apply, anywhere, ever. 

IFF your intention was to 'temporarily' disable the rule then you should add the 'exception' to the ruleset.json file. 

The effect will be the same, the intent you indicate quite different.

Of course I am still of the opinion the rule should not be disabled at all, not even temporarily.